pkg/cmd/account/list/list.go

```diff
@@ -32,17 +36,25 @@ func NewCmdList(client factory.Factory) *cobra.Command {
 				Id   string
 				Name string
 				Type string
-				// TODO should we list type-specific fields here? (e.g. an Azure DevOps account may have different attributes than a Username)
 			}
 
-			return output.PrintArray(allAccounts, cmd, output.Mappers[accounts.IAccount]{
+			accountTypeMap := map[accounts.AccountType]string{
```

Comment:
I wonder if the Go client would ultimately be a better place for this to live? Doesn't have to be for now, maybe we should consider as for v3 of the client?

Reply:
100% we should consider moving this at some point.